FordLabs/PeopleMover
# Update react scripts to fix security vulnerability for object-path package

## What was done
- [x] Fixes dependabot security alert caused by object-path 

## How to test
1. Nothing should change in app functionally

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
